.travis.yml

```diff
@@ -52,6 +52,7 @@ script:
   - if [ "$TRAVIS_OS_NAME" = "linux" ]; then cd $TRAVIS_BUILD_DIR && python3 tests/integration_tests/flows/test_clickhouse.py; fi
   - if [[ "$TRAVIS_OS_NAME" = "linux" && $(lsof -t -i:47334) ]]; then kill -9 $(lsof -t -i:47334); fi
   - if [ "$TRAVIS_OS_NAME" = "linux" ]; then ps -aux | grep $(lsof -t -i:47334) || true; fi
+  - if [ "$TRAVIS_OS_NAME" = "linux" ]; then sleep 61; fi
```

Comment:
i have tried it before, no results, 'address already' in use still appears some time. I added some more 'process kill' stuff in previous pr, may be it will help